src/CMakeLists.txt

```diff
@@ -767,6 +767,12 @@ if(WITH_TESTS)
     find_package(GTest 1.13.0 REQUIRED)
     find_package(GMock REQUIRED)
   else()
+    if(NOT COMPILER_IGNORES_DEPRECATED_DECL_IN_SYSTEM_HEADERS)
+      # See https://github.com/llvm/llvm-project/issues/76515
+      set_property(DIRECTORY googletest
+        APPEND "-Wno-deprecated-declarations"
```

Comment:
Some `make check` jobs are now failing with:
```
CMake Error at src/CMakeLists.txt:772 (set_property):
  set_property given invalid argument "-Wno-deprecated-declarations".
```
https://jenkins.ceph.com/job/ceph-pull-requests/155301/console

In particular, irvingi07 worker seems to be affected.

Reply:
Sorry for the regression, will take a look ASAP.

Reply:
should be fixed by #62641